rtsf-at-checkout-rsp-controller-event-handler/eventhandler/process_events.go

```diff
@@ -18,61 +17,52 @@ const (
 	RFIDEventMoved    = "moved"
 	ROIActionEntered  = "ENTERED"
 	ROIActionExited   = "EXITED"
-)
 
-var loggingClient logger.LoggingClient
+	resourceName = ""
+	sourceName   = "rfid-roi-event"
+	deviceName   = "device-rfid-roi-rest"
```

Comment:
This will need to change to be name appropriately. Something like `rfid-roi-rest`. 
Can wait until the Device REST profile and device definition are migrated.

Reply:
I have made the suggested change